drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/resources/assets/dmn-list-composite.dmn

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" ?>
```

Comment:
Is it a test resource? If yes please move to test resource folder

Reply:
@danielezonca These dmn files are already available in test/resources folder.After a discussion with @gitgabrio, we decided to remove them.